pkg/repo/types.go

```diff
@@ -0,0 +1,62 @@
+package repo
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"time"
+)
+
+const (
+	timeformat = "2006-01-02 15:04:05"
+	writeperm  = 0777
+)
+
+type RepoList struct {
+	Sources []RepoEntry `yaml:"sources"`
+}
+
+type RepoEntry struct {
+	Name string `yaml:"name"`
+	Url  string `yaml:"url"`
+	Date string `yaml:"updated"`
```

Comment:
It's not totally clear to me the purpose of storing the date here. Could you explain it, please?

Reply:
Falcoctl does not rely on `Date` for the logic of its commands: it's just extra info to print for the user, represents the last time this entry was fetched/updated. Do you think it's unnecessary? Thanks!